packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx

```diff
@@ -0,0 +1,8 @@
+import {render} from "@testing-library/react"
+import React from "react"
+import {Search} from "."
+
+test("renders ApiPane snapshot", () => {
+    const {asFragment} = render(<Search getSearchResults />)
+    expect(asFragment()).toMatchSnapshot()
```

Comment:
The tests aren't running? And the snapshsot not checked in? Still see old Facebook tests.

Reply:
I still have not touches any testing files. As we have an outstanding task on the board. @estherthetesteramido

Reply:
Fair. Can we instead then delete the files if they aren't being used?